openedx_lti_tool_plugin/deep_linking/tests/test_forms.py

```diff
@@ -0,0 +1,106 @@
+
+    @patch(f'{MODULE_PATH}.forms.MultipleChoiceField')
+    @patch(f'{MODULE_PATH}._')
+    @patch.object(DeepLinkingForm, 'get_content_items_choices')
+    def test_init(
+        self,
+        get_content_items_choices_mock: MagicMock,
+        gettext_mock: MagicMock,
+        multiple_choice_field_mock: MagicMock,
+    ):
+        """Test `__init__` method."""
+        self.assertEqual(
+            self.form_class(request=self.request).fields,
+            {'content_items': multiple_choice_field_mock.return_value},
+        )
+        get_content_items_choices_mock.assert_called_once_with(self.request)
+        gettext_mock.assert_called_once_with('Courses')
+        multiple_choice_field_mock.assert_called_once_with(
+            choices=get_content_items_choices_mock(),
+            required=False,
+            widget=forms.CheckboxSelectMultiple,
+            label=gettext_mock(),
+        )
+
+    @patch(f'{MODULE_PATH}.course_context')
+    @patch.object(DeepLinkingForm, 'get_content_items_choice')
+    @patch.object(DeepLinkingForm, '__init__', return_value=None)
+    def test_get_content_items_choices(
+        self,
+        deep_linking_form_init: MagicMock,  # pylint: disable=W0613
+        get_content_items_choice_mock: MagicMock,
+        course_context_mock: MagicMock,
+    ):
+        """Test `get_content_items_choices` method."""
+        course_context_mock.return_value.objects.all.return_value = [self.course]
+
+        self.assertEqual(
+            self.form_class().get_content_items_choices(self.request),
+            [get_content_items_choice_mock.return_value],
+        )
+        course_context_mock.assert_called_once_with()
+        course_context_mock().objects.all.assert_called_once_with()
+        get_content_items_choice_mock.assert_called_once_with(self.course, self.request)
+
+    @patch(f'{MODULE_PATH}.reverse')
+    @patch.object(DeepLinkingForm, '__init__', return_value=None)
+    def test_get_content_items_choice(
+        self,
+        deep_linking_form_init: MagicMock,  # pylint: disable=W0613
+        reverse_mock: MagicMock,
+    ):
+        """Test `get_content_items_choice` method."""
+        self.assertEqual(
+            self.form_class().get_content_items_choice(self.course, self.request),
+            (
+                self.request.build_absolute_uri.return_value,
+                self.course.learning_context.title,
+            ),
+        )
+        reverse_mock.assert_called_once_with(
+            f'{app_config.name}:1.3:resource-link:launch-course',
+            kwargs={'course_id': self.course.learning_context.context_key},
+        )
+        self.request.build_absolute_uri.assert_called_once_with(reverse_mock())
+
+    @patch(f'{MODULE_PATH}.DeepLinkResource')
+    @patch.object(DeepLinkingForm, '__init__', return_value=None)
+    def test_get_deep_link_resources(
+        self,
+        deep_linking_form_init: MagicMock,  # pylint: disable=W0613
```

Comment:
@kuipumu Please use the explicit pylint title.

```suggestion
        deep_linking_form_init: MagicMock,  # pylint: disable=unused-argument
```